crawl-ref/source/dat/des/altar/altar.des

```diff
@@ -384,6 +450,51 @@ cccyy..yycc
      @@
 ENDMAP
 
+# Using the 'can_overwrite' tag to simulate Lugonu corrupting the dungeon
+# validate possibly not needed, but better safe than sorry to check that
+# we can still traverse whatever Lugonu has chosen to put their foot in.
+# The use of this many @s is also probably overkill, but want to take
+# all measures I can think of to stop can_overwrite messing up level
+# traversal with this vault since it can overwrite.
+NAME:     amcnicky_altar_lugonu_corruption
+TAGS:     transparent can_overwrite no_monster_gen
```

Comment:
Thanks, I'm going to play around with a few versions of "pre-built corruption" in that case and see how it comes up in-game and go from there.

Reply:
@ebering can_overwrite is from in here! https://crawl.develz.org/wiki/doku.php?id=dcss:help:maps:syntax:headers. Tagging you in this comment in particular in case that needs changing, especially if it doesn't actually work in the current codebase.

Reply:
Think I've landed on a better 12x12 version of this for submission, now moving on. Looking into a corrupted level layout engine in future is a super interesting idea. I'm going to walk before I can run for now though.